x/staking/src/types/domain.rs

```diff
@@ -500,3 +501,82 @@ impl From<Validator> for ValidatorRaw {
 }
 
 impl Protobuf<ValidatorRaw> for Validator {}
+
+/// [`Validators`] is a collection of [`Validator`] with some guarantees:
+/// - the collection cannot have duplicated validators by public key
+/// - no validator can be bonded and jailed at the same time // TODO: should this be a property of the validator itself?
+/// - no bonded/unbonded validator can have zero delegator shares // TODO: should this be a property of the validator itself?
+#[derive(Debug, Clone, Deserialize, Serialize, Default)]
+#[serde(try_from = "Vec<Validator>")]
+pub struct Validators(Vec<Validator>);
```

Comment:
Small comment. We need to rename `SendCoins` to `Coins`.

Reply:
Yes, agreed. We have two types for coin collections, we should give them compatible names.